tests/ui/impl-trait/erased-regions-in-hidden-ty.rs

```diff
@@ -0,0 +1,21 @@
+// revisions: current next
+// compile-flags: -Zverbose
+//[next] compile-flags: -Ztrait-solver=next
+// normalize-stderr-test "DefId\([^\)]+\)" -> "DefId(..)"
+
+#![feature(rustc_attrs)]
+#![rustc_hidden_type_of_opaques]
+
+// Make sure that the compiler can handle `ReErased` in the hidden type of an opaque.
+
```

Comment:
pls add a comment that you can't write the whole type name as it contains a `DefId`